introduction.md

```diff
@@ -1,2 +1,7 @@
 Hello, my name is Brian.

```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?